tests/checker/CidChecker.spec.ts

```diff
@@ -185,7 +185,7 @@ To apply for DataCap to onboard your dataset to Filecoin, please fill out the fo
     });
   })
 
-  describe('getLocation', () => {
+  xdescribe('getLocation', () => {
```

Comment:
I've fixed this one. It works in CI because the IPINFO_TOKEN is set as a test secret, but not locally. Fixed by changing the token to empty string instead of `undefined`